packages/core/src/state-management/__tests__/cache-eviction.test.ts

```diff
@@ -7,10 +7,12 @@ import { delay } from '../../pubsub/__tests__/delay';
 
 let ipfs: IpfsApi;
 let ceramic: Ceramic;
+let metadata: any;
 
 beforeAll(async () => {
   ipfs = await createIPFS();
   ceramic = await createCeramic(ipfs, { streamCacheLimit: 1, anchorOnRequest: false });
+  metadata = { deterministic: true, controllers: [ceramic.did.id], family: "family", tags: ["x", "y"] }
```

Comment:
If more than 1 test uses the same deterministic metadata to create streams then the actions of one test can have impact on the other tests.  I think each test that needs to use deterministic document lookups should make their own unique metadata.  Could add a helper for generating the metadata to make that cleaner though

Reply:
Ok, even though we're creating fresh Ceramic/IPFS instances before each test and closing them after? Is it because of the parallelism of the test execution?

Reply:
We don't create a new ceramic for each test.  `beforeAll`/`afterAll` run once per suite. `beforeEach`/`afterEach` run for each test, but that's not what we're using here.

Reply:
Darn, you're right. I always get confused by the names.

Reply:
Based on [this](https://github.com/ceramicnetwork/js-ceramic/pull/1392#discussion_r638117147) comment, I just removed `metadata` and used `TileDocument.load()` with non-deterministic documents.